<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1001</width>
    <height>732</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="LBL_AuxChan1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Ch1</string>
    </property>
   </widget>
   <widget class="QLabel" name="LBL_MessageNo1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Message No</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_MessageNo1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="LBL_MemoryType1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Memory Type</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_MemoryType1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="LBL_Duration1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Duration</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_Duration1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CBX_Enable1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Enabled</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CBX_Input1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>150</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Input</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_Change1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Change</string>
    </property>
   </widget>
   <widget class="QLabel" name="LBL_MessageNo2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Message No</string>
    </property>
   </widget>
   <widget class="QLabel" name="LBL_AuxChan2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Ch2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_MemoryType2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>60</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_Duration2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>110</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CBX_Input2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>150</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Input</string>
    </property>
   </widget>
   <widget class="QLabel" name="LBL_Duration2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>90</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Duration</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_MessageNo2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_Change2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>150</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Change</string>
    </property>
   </widget>
   <widget class="QLabel" name="LBL_MemoryType2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>40</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Memory Type</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CBX_Enable2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>110</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Enabled</string>
    </property>
   </widget>
   <widget class="QLabel" name="LBL_MessageNo3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Message No</string>
    </property>
   </widget>
   <widget class="QLabel" name="LBL_AuxChan3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Ch3</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_MemoryType3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>250</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_Duration3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CBX_Input3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>340</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Input</string>
    </property>
   </widget>
   <widget class="QLabel" name="LBL_Duration3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Duration</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_MessageNo3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_Change3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Change</string>
    </property>
   </widget>
   <widget class="QLabel" name="LBL_MemoryType3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>230</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Memory Type</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CBX_Enable3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>300</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Enabled</string>
    </property>
   </widget>
   <widget class="QLabel" name="LBL_MessageNo4">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>230</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Message No</string>
    </property>
   </widget>
   <widget class="QLabel" name="LBL_AuxChan4">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>200</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Ch4</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_MemoryType4">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>250</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_Duration4">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>300</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CBX_Input4">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>340</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Input</string>
    </property>
   </widget>
   <widget class="QLabel" name="LBL_Duration4">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>280</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Duration</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_MessageNo4">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>250</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_Change4">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>340</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Change</string>
    </property>
   </widget>
   <widget class="QLabel" name="LBL_MemoryType4">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>230</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Memory Type</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CBX_Enable4">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>300</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Enabled</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_MemoryType_Main">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>430</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QLabel" name="LBL_MemoryType_Main">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>410</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Memory Type</string>
    </property>
   </widget>
   <widget class="QLabel" name="LBL_MessageNo_Main">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Message No</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_MessageNo_Main">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_RunMessage">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Run Message</string>
    </property>
   </widget>
   <widget class="QSlider" name="SLD_Time">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>590</y>
      <width>861</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000000</number>
    </property>
    <property name="singleStep">
     <number>100</number>
    </property>
    <property name="pageStep">
     <number>500</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_Time">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>610</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_Blank">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>470</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Blank Sign</string>
    </property>
   </widget>
   <widget class="QLabel" name="LBL_Current_Display">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>20</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Display</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_MessageNo_Current">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>70</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="LBL_MessageNo_Current">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>50</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Message No</string>
    </property>
   </widget>
   <widget class="QLabel" name="LBL_MemoryType_Current">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>50</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Memory Type</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_MemoryType_Current">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>70</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_Reset">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>650</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1001</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
